subprojects/core/src/main/java/org/gradle/api/internal/project/DefaultProject.java

```diff
@@ -145,6 +146,7 @@
 
     private final ClassLoaderScope classLoaderScope;
     private final ClassLoaderScope baseClassLoaderScope;
+    private final ProtectApiService protectApiService;
```

Comment:
⭕️ This name really doesn't tell me what this class is for. Should probably put this logic in the existing [`CrossProjectConfigurator`.](https://github.com/gradle/gradle/blob/master/subprojects/core/src/main/java/org/gradle/api/internal/project/CrossProjectConfigurator.java)

Reply:
Thanks Stefan, I didn't know about the `CrossProjectConfigurator`